cli/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "nickel-lang-cli"
+version = "1.0.0"
+authors = ["The Nickel Team <nickel-lang@protonmail.com>"]
+description = "Common helper functions for the tests and benchmarks of the nickel-lang crate."
+edition = "2021"
+
+[[bin]]
+name = "nickel"
+path = "bin/nickel.rs"
+bench = false
+
+[lib]
```

Comment:
Do we need a lib here, if it's only the binary?

Reply:
That was mostly for future topiary integration where I didn't want to continue cluttering `bin/nickel.rs`.